release_notes/addtl-release-notes.adoc

```diff
@@ -0,0 +1,96 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="addtl-release-notes"]
+= Additional release notes
+include::_attributes/common-attributes.adoc[]
+include::_attributes/attributes-microshift.adoc[]
+include::_attributes/servicebinding-document-attributes.adoc[]
+:context: addtl-release-notes
+
+toc::[]
+
+Release notes for additional related components and products not included in the core xref:../release_notes/ocp-release-notes.adoc#ocp-release-notes[{product-title} {product-version} release notes] are available in the following documentation.
+
+[IMPORTANT]
+====
+The following release notes are for downstream Red Hat products only; upstream or community release notes for related products are not included.
+====
+
+[cols="1a,1a,1a",frame=none,grid=none]
+|===
+|A::
+xref:../networking/aws_load_balancer_operator/aws-load-balancer-operator-release-notes.adoc#aws-load-balancer-operator-release-notes[AWS Load Balancer (ALB) Operator]
```

Comment:
ALB or ALBO:

See [example](https://docs.redhat.com/en/documentation/red_hat_openshift_service_on_aws/4/html-single/networking/index#aws-uninstalling-an-aws-load-balancer-operator_aws-load-balancer-operator).

Consider dropping the acronym. AWS docs do not commonly use the acronym.